tests/queries/0_stateless/01802_dict_sparse_hashed_bytes_allocated.sql

```diff
@@ -0,0 +1,29 @@
+DROP DATABASE IF EXISTS 01802_db;
+CREATE DATABASE 01802_db;
```

Comment:
Don't create databases explicitly. Use `currentDatabase()` or don't use databases at all. This is for parallel execution of a tests, where databases are generated automatically and randomly.

Reply:
Interesting, before there was some issues with dictionaries, and they requires explicit database, but right now everything works, will update the test